tests/unit/test_venv.py

```diff
@@ -433,6 +433,10 @@ def test_install_command_not_installed(newmocksession):
 
 
 def test_install_command_whitelisted(newmocksession):
+    """
+    Deprecated: Remove in favor of test_install_command_allowlisted when
```

Comment:
Remove this docstring, the migration complete will not happen before the rewrite.